easybuild/easyconfigs/p/PGPLOT/PGPLOT-5.2.2-GCCcore-10.3.0.eb

```diff
@@ -34,15 +34,21 @@ dependencies = [
     ('X11', '20210518'),
 ]
 
-cmds_map = [('.*', './makemake . linux g77_gcc && make && make cpg')]
+cmds_map = [('.*', './makemake . linux g77_gcc && make && make shared && make cpg && make cpg-shared')]
 
 files_to_copy = [
     (['pgdemo*'], 'bin'),
-    (['libpgplot.a', 'libcpgplot.a'], 'lib'),
+    (['libpgplot.*', 'libcpgplot.*'], 'lib'),
     (['grfont.dat'], 'share'),
     (['cpgplot.h'], 'include'),
 ]
 
+postinstallcmds = [
+    # create symlinks
+    "cd %(installdir)s/lib && ln -s libcpgplot.so.5.2.2 libcpgplot.so.5 ; ln -s libcpgplot.so.5.2.2 libcpgplot.so && "
+    " ln -s libpgplot.so.5.2.2 libpgplot.so.5 ; ln -s libpgplot.so.5.2.2 libpgplot.so"
+]
+
 sanity_check_paths = {
     'files': ['lib/libpgplot.a', 'lib/libcpgplot.a'],
```

Comment:
Since you're now also building the shared libraries, the sanity check should also check they exist.